app/models/media.rb

```diff
@@ -51,11 +51,21 @@ def alt
   end
 
   def thumbnail?
-    kind_value == 'image'
+    kind_value == 'image' ? (item_content_type.include? "image") : false
   end
 
   def recalculate_loan_health
     # Only submit a job if it is a Loan
     RecalculateLoanHealthJob.perform_later(loan_id: media_attachable_id) if media_attachable_type == 'Project'
   end
+
+  def visual_media?
+    if kind_value == "image"
```

Comment:
How about `%w(image video).include?(kind_value)`?

Also how about changing the name to just `visual?` because the media part is redundant?